### DISCUSSION TITLE
[bug]: node fall down, and I suppose that it is a common problem.

### DISCUSSION THREAD

**Discussion by ybaidiuk** [lightningnetwork/lnd#8034] [Troubleshooting]
Today 1h ago my node get error. 
![image](https://github.com/lightningnetwork/lnd/assets/11969255/31fdc5f1-9e65-4d79-ba94-989e8c5788d3)
i downloaded logs and restarted the node. 
Interesting is that after getting online again, 10% of my peers were offline, and it looks that they're running the same software.
[umbrel-1695159776827-debug.log](https://github.com/lightningnetwork/lnd/files/12666878/umbrel-1695159776827-debug.log)
![Screenshot 2023-09-19 at 23 56 13](https://github.com/lightningnetwork/lnd/assets/11969255/ee2e0f4f-7978-4030-82f8-8d7011bb1f37)

**Comment by bitromortac** [comment]
The logs you attached unfortunately don't provide any insights of what could have crashed the node. It would be helpful if you could provide logs for lightning-terminal (litd) and/or lnd during the crash.

**Comment by saubyk** [comment]
Moving this issue to a discussion thread as it appears to be a troubleshooting request
